Prototyper/Classes/Login/LoginView.swift

```diff
@@ -0,0 +1,128 @@
+//
+//  LoginView.swift
+//  KeychainSwift
+//
+//  Created by Raymond Pinto on 01.12.19.
+//
+
+import SwiftUI
+
+struct LoginView: View {
```

Comment:
I would suggest to always show the `LoginView` using a modal presentation. This allows us to keep it independent of whatever view comes after it. It should be able to change some global state e.g. in the APIHandler or PrototyperSetting and then dismiss itself after its work is done. All other views can observe that shared state and can react accordingly. This would make it easier to encapsulate the logic and make this view more generic.